extension/EdFiLMSMetaEd/DomainEntity/Assignment.metaed

```diff
@@ -43,4 +43,7 @@ Domain Entity Assignment
         role name Max
     domain entity EdFi.Section
         documentation "Reference to an Section in the standard Ed-Fi data model."
-        is optional
+        is required
+    domain entity EdFi.School
+        documentation "Reference to a School in the standard Ed-Fi data model."
+        is required
```

Comment:
Added this as a guess to see if it would resolve the following error when trying to PUT a new Assignment. It did not help.

```
2021-06-19 16:04:29,830 [48] ERROR AssignmentsController - Post
System.NotSupportedException: Relationship-based authorization could not be performed on the request because there were no authorization segments defined indicating the resource shouldn't be authorized with a relationship-based strategy.
   at EdFi.Ods.Security.AuthorizationStrategies.Relationships.RelationshipsAuthorizationStrategyBase`1.AuthorizeSingleItemAsync(IEnumerable`1 relevantClaims, EdFiAuthorizationContext authorizationContext, CancellationToken cancellationToken) in C:\source\ed-fi\Ed-Fi-ODS\Application\EdFi.Ods.Security\AuthorizationStrategies\Relationships\RelationshipsAuthorizationStrategyBase.cs:line 141
   at EdFi.Ods.Security.Authorization.EdFiAuthorizationProvider.AuthorizeSingleItemAsync(EdFiAuthorizationContext authorizationContext, CancellationToken cancellationToken) in C:\source\ed-fi\Ed-Fi-ODS\Application\EdFi.Ods.Security\Authorization\EdFiAuthorizationProvider.cs:line 123
   at EdFi.Ods.Security.Authorization.Repositories.RepositoryOperationAuthorizationDecoratorBase`1.AuthorizeSingleItemAsync(T entity, String actionUri, CancellationToken cancellationToken) in C:\source\ed-fi\Ed-Fi-ODS\Application\EdFi.Ods.Security\Authorization\Repositories\RepositoryOperationAuthorizationDecoratorBase.cs:line 64
   at EdFi.Ods.Security.Authorization.Repositories.CreateEntityAuthorizationDecorator`1.CreateAsync(T entity, Boolean enforceOptimisticLock, CancellationToken cancellationToken) in C:\source\ed-fi\Ed-Fi-ODS\Application\EdFi.Ods.Security\Authorization\Repositories\CreateEntityAuthorizationDecorator.cs:line 60
   at EdFi.Ods.Common.Infrastructure.Repositories.UpsertEntity`1.UpsertAsync(TEntity entity, Boolean enforceOptimisticLock, CancellationToken cancellationToken) in C:\source\ed-fi\Ed-Fi-ODS\Application\EdFi.Ods.Common\Infrastructure\Repositories\UpsertEntity.cs:line 72
   at EdFi.Ods.Security.Authorization.Repositories.UpsertEntityAuthorizationDecorator`1.UpsertAsync(T entity, Boolean enforceOptimisticLock, CancellationToken cancellationToken) in C:\source\ed-fi\Ed-Fi-ODS\Application\EdFi.Ods.Security\Authorization\Repositories\UpsertEntityAuthorizationDecorator.cs:line 41
   at EdFi.Ods.Api.Infrastructure.Pipelines.Steps.PersistEntityModel`4.ExecuteAsync(TContext context, TResult result, CancellationToken cancellationToken) in C:\source\ed-fi\Ed-Fi-ODS\Application\EdFi.Ods.Api\Infrastructure\Pipelines\Steps\PersistEntityModel.cs:line 36
```

Reply:
I'm going to defer this issue until LMS-318 so that these descriptor changes can move forward.